.github/pull_request_template.md

```diff
@@ -1,6 +1,3 @@
-This is the link to the project: [TITLE](URL)
```

Comment:
This is important for reviewers, so we don't have to copy/paste urls from the PR files.

Reply:
Got no strong opinion, but I don't think it's a big deal to select and copy-paste text. Could help catch typos if you're forced to use the actual URL

Reply:
This should stay. It was specifically requested by Daniel and It would help me check out the project faster on mobile.

Reply:
Good point!